rt/ws/rm/src/main/java/org/apache/cxf/ws/rm/persistence/jdbc/RMTxStore.java

```diff
@@ -886,6 +886,8 @@ protected void verifyTable(Connection con, String tableName, String[][] tableCol
             while (rs.next()) {
                 dbCols.add(rs.getString(4));
             }
+            rs.close();
```

Comment:
Same, could we please wrap in it a `try` block? thank you

Reply:
Wrapped in try